akka-docs-dev/rst/scala/code/docs/stream/io/StreamFileDocSpec.scala

```diff
@@ -32,6 +32,7 @@ class StreamFileDocSpec extends AkkaSpec(UnboundedMailboxConfig) {
     //#file-source
     import akka.stream.io._
     //#file-source
+    Thread.sleep(0)
```

Comment:
There needs to be a statement here, otherwise scalariform chokes on the (admittedly also invalid) syntax. Will add a comment.